src/main/java/uk/gov/hmcts/juror/api/bureau/service/UserServiceImpl.java

```diff
@@ -417,21 +417,9 @@ public AssignmentsListDto getStaffAssignments(AssignmentsMultiRequestDto respons
             String jurorFirstName = jurorResponse.getFirstName();
             String jurorLastName = jurorResponse.getLastName();
 
-            jurorDisplayName.append((jurorTitle != null)
-                ?
-                jurorTitle
-                :
-                    "");
-            jurorDisplayName.append((jurorFirstName != null)
-                ?
-                " " + jurorFirstName
-                :
-                    "");
-            jurorDisplayName.append((jurorLastName != null)
-                ?
-                " " + jurorLastName
-                :
-                    "");
+            jurorDisplayName.append((jurorTitle != null) ? jurorTitle : "");
+            jurorDisplayName.append((jurorFirstName != null) ? " " + jurorFirstName : "");
+            jurorDisplayName.append((jurorLastName != null) ? " " + jurorLastName : "");
```

Comment:
Enclosing braces not needed i.e 
((jurorTitle != null) ? jurorTitle : "")
should be
(jurorTitle != null ? jurorTitle : "")

etc

Reply:
Also need to look when we prepend the space.

As if title is null we do not need the space before juror First Name, etc

Reply:
removed brackets